source/includes/_plaid_accounts.md

```diff
@@ -2,24 +2,25 @@
-| last_import                  | string | Timestamp in ISO 8601 extended format of the last time Lunch Money imported new data from Plaid for this account.                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| last_fetch                   | string | Timestamp in ISO 8601 extended format of the last successful check from Lunch Money for updated data or timestamps from Plaid in ISO 8601 extended format (not necessarily date of last successful import)                                                                                                                                                                                                                                                                                                                                             |
-| plaid_last_successful_update | string | Timestamp in ISO 8601 extended format of the last time Plaid successfully connected with institution for new transaction updates, regardless of whether any new data was available in the update.                                                                                                                                                                                                                                                                                                                                                      |
+Attribute Name                | Type   | Nullable | Description
+----------------------------- | ------ | -------- | ------------------------------------------------------------------------------------------------------------------
+id                            | number | false    | Unique identifier of Plaid account
+date_linked                   | string | false    | Date account was first linked in ISO 8601 format
+name                          | string | false    | Name of the account. Can be overridden by the user. Field is originally set by Plaid
+display_name                  | string | false    | Display name of the account, if not set it will return an empty string.
+type                          | string | false    | Primary type of account. Typically one of:<br><ul> <li>credit</li> <li>depository</li> <li>brokerage</li> <li>cash</li> <li>loan</li> <li>Investment</li><ul><br> This field is set by Plaid and cannot be altered
+subtype                       | string | true     | Optional subtype name of account. This field is set by Plaid and cannot be altered
```

Comment:
Wow, I just checked and there are only 71 accounts out of 55000 with subtype = null

Reply:
Wow, well perfect that one of those accounts in mine so that I could label this correctly haha